script/proof

```diff
@@ -32,4 +32,4 @@ bundle exec jekyll build -s $SOURCE -d $DESTINATION --trace
 
 # 3.
 msg "Proofing..."
-time bundle exec htmlproofer ./$DESTINATION --url-ignore $INGORE_HREFS $@
+time bundle exec htmlproofer ./$DESTINATION --allow-hash-href --disable-external --url-ignore $IGNORE_HREFS $@
```

Comment:
`script/proof` is not like our normal CI that needs to pass no matter what, so it's ok to leave it half broken. I'd say remove `--disable-external` from this PR.